recipes/knit/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.2.1" %}
+
+package:
+    name: knit
+    version: {{version}}
+
+source:
+    git_url: https://github.com/dask/knit   
+    git_rev: {{version}}
```

Comment:
Please do not use git as the source, instead use a tarball. See docs https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos